src/braket/pennylane_plugin/ahs_device.py

```diff
@@ -158,7 +158,7 @@ def generate_samples(self):
         Returns:
              array[complex]: array of samples in the shape ``(dev.shots, dev.num_wires)``
         """
-        return [self._result_to_sample_output(res) for res in self.samples.measurements]
+        return np.array([self._result_to_sample_output(res) for res in self.samples.measurements])
 
     def _validate_operations(self, operations):
```

Comment:
I think we should include an `expval` method on this device that adds basis checks and raises appropriate errors before calling the parent `expval` method as normal, otherwise some things will 'succeed', but not really.